spec/registrant_spec.rb

```diff
@@ -0,0 +1,41 @@
+require './lib/registrant'
+
+Rspec.describe Registrant do
+    it 'the registrant has a name' do
+        registrant_1 = Registrant.new('Bruce', 18, true )
+        registrant_2 = Registrant.new('Penny', 15 )
+
+        expect(registrant_1.name).to eq("Bruce")
+    #    expect(registrant_1.age).to eq(18)
+     #   expect(registrant_1.permit).to eq(true)
+        expect(registrant_2.name).to eq("Penny")
+      #  expect(registrant_2.age).to eq(15)
+      #  expect(registrant_2.permit).to eq(false)
+    end
+
+    it 'has an age' do
+        registrant_1 = Registrant.new('Bruce', 18, true )
+        registrant_2 = Registrant.new('Penny', 15 )
+
+        expect(registrant_1.age).to eq(18)
+        expect(registrant_2.age).to eq(15)
+    end
+
+    it 'has a permit?' do
+        registrant_1 = Registrant.new('Bruce', 18, true )
+        registrant_2 = Registrant.new('Penny', 15 )
+
+        expect(registrant_1.permit).to eq(true)
```

Comment:
A permit? method is defined in the interaction pattern, and is what should be tested here